app/src/main/java/com/example/paritosh/stormy/WeatherContract.java

```diff
@@ -0,0 +1,32 @@
+package com.example.paritosh.stormy;
+
+
+import android.content.Context;
+import android.support.annotation.StringRes;
+
+import com.example.paritosh.stormy.model.CurrentWeatherDataBindingModel;
+
+public interface WeatherContract {
+
+    interface WeatherView {
+
+        void render(CurrentWeatherDataBindingModel model);
+
+        void showMessage(@StringRes int messageResId);
+
+        void errorMessage(@StringRes int titleResId, @StringRes int messageResId);
+
+        Context getContext();
```

Comment:
Sorry, I didn't realize that get is misleading as only getters in java start with "get". Maybe retrieveContext or loadContext would be appropriate.